src/Controller/UserController.php

```diff
@@ -317,6 +324,7 @@ public function deleteUser(User $user, UserRepository $userRepository): JsonResp
                 'message' => 'Access denied!'
             ], Response::HTTP_FORBIDDEN);
         }
+        $this->commentTransformer->clearUserComments($authenticatedUser);
```

Comment:
This should be called by `$userRepository->delete()`. Like that you won't forget about clearing user comments in other places.